legal-api/report-templates/template-parts/common/addresses.html

```diff
@@ -0,0 +1,87 @@
+<div class="address-container">
+   {% if offices.registeredOffice is defined %}
+   <div class="section-title">
+      Registered Office Information
+   </div>
+   <div class="section-data">
+      <table class="section-data-table" role="presentation">
+         <tr>
+            <td class="delivery-address">
+               <div class="section-sub-title">Delivery Address</div>
+               <div>{{ offices.registeredOffice.deliveryAddress.streetAddress }}</div>
+               <div>{{ offices.registeredOffice.deliveryAddress.streetAddressAdditional }}</div>
+               <div>{{ offices.registeredOffice.deliveryAddress.addressCity }}
```

Comment:
Per comments in /bcgov/entity#4230, the city and region should be on 1 line, and the postal code and country should be on the next line.

This applies to all addresses.

Reply:
Will not be addressing formatting stuff in this PR. This is mainly about framework change and NOA

Reply:
OK. Please keep track of this for a future PR.